lite/jupyter-lite.json

```
@@ -45,7 +45,7 @@
         "startMode": "single"
       },
       "@jupyterlab/notebook-extension:tracker": {
-        "autoStartDefaultKernel": true,
+        "autoStartDefaultKernel": false,
```

Switching this back to `true`; this will be changed in a follow-up PR.

```suggestion
        "autoStartDefaultKernel": true,
```